concepticondata/conceptlists/Jonauskaite-2020-12.tsv

```diff
@@ -1,13 +1,13 @@
-ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ARABIC	AZERBAIJANI	CHINESE	DUTCH	ESTONIAN	FINNISH	FRENCH	GERMAN	GEORGIAN	GREEK	HEBREW	ITALIAN	LITHUANIAN	NORWEGIAN	PERSIAN	POLISH	RUSSIAN	SERBIAN	SPANISH	SWEDISH	UKRAINIAN
```

Comment:
The column heading for Arabic was changed to Persian. In the original list in the PDF, it is "Arabic". We also have a glossing language Arabic. Was there a reason why it was changed?

Reply:
Good catch. Can you please find out if it is Arabic or Persian?

Reply:
I'm not really sure how this has happened; the Arabic column has Arabic and the Persian has Persian.

Reply:
Maybe that was another mistake in the PDF?

Reply:
Judging from a spot check with Google Translate "Detect language" the entries in the column Persian belong to Persian and the entries in the column Arabic belong to Arabic.  Not sure what caused the highlighting of a change.